### ISSUE TITLE
Banner removed on refresh

### ISSUE THREAD

**Issue by jenstornell** [Alex-D/Cookies-EU-banner#39]
If I don't accept or deny the banner but instead refresh the page, the banner is gone. So now my GA code runs even without approving the cookies.

What I noticed what that except the ga cookies, there was `hasConsent: true`. Not sure if that has something to do with it?

**Comment by Alex-D** [comment]
Yep, it was OK before RGPD but it has changed a little bit, so now we should wait the user click on something before consider he navigates.
